Dynatrace/dynatrace-operator
# Rename LogAgent to LogModule

## Description

Just to conform with the naming.
Naming changed to `LogModule` after initial ticket creation.

re-Generated:
- deepcopy
- api-ref
- crd

## How can this be tested?
Not much. Check the naming and doc fields.
Can check if I committed everything:
- `make manifests`
- `make doc/api-ref`
- `make manifests/deepcopy`
